deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml

```diff
@@ -80,6 +80,10 @@ spec:
                   type: string
                 incremental:
                   type: boolean
+                mavenMirrorURL:
+                  description: MavenMirrorUrl Maven Mirror to be used during source-to-image
```

Comment:
I guess description doesn't have to repeat `MavenMirrorUrl`.

Reply:
+1

Reply:
it is updated during CRD generator. @ricardozanini ^

Reply:
hmm, during the CRD generation the comment is read. :)